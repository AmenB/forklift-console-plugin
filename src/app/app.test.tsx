// TODO replace enzyme with react-testing-library!
describe('App tests', () => {
  test('noop -- TODO: add real tests!', () => {
    expect(true).toBe(true);
  });
});
