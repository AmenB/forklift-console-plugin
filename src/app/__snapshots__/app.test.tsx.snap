// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App tests should render default App component 1`] = `
<BrowserRouter>
  <LocalStorageContextProvider>
    <AppLayout>
      <Component />
    </AppLayout>
  </LocalStorageContextProvider>
</BrowserRouter>
`;
